src/RoyalMailMethod.php

```diff
@@ -0,0 +1,56 @@
+<?php namespace Meanbee;
+
+/**
```

Comment:
No need for this license text in every file, we can just do it in a `LICENSE.md` in the root of the repository.